doc/source/tutorials/tutorial-level-5-migration.rst

```diff
@@ -297,8 +315,8 @@ Move essential files to run local tests
 
     .. code-block:: bash
 
-        $ cp -n -r ../src .
-        $ cp -n -r ../tests .
+        $ cp -n -r ../src src/
```

Comment:
<img width="435" alt="Screenshot 2025-06-12 at 7 06 16 AM" src="https://github.com/user-attachments/assets/3caaf7ed-6cae-4965-b149-fc34dd0fe571" />

are these `src/` and `test/s` needed? Above are commands I use all the time for recutting.

Reply:
I left a comment below.  There are still two entries in the matrix that need testing....i.e., when `./tests/` doesn't exist.  I guess this is unlikely because it should have been created by skpkg, but it is helpful to just complete the test to understand everything.